.gitignore

```diff
@@ -34,3 +34,7 @@
 /docs/.firebaserc
 /node_modules
 /frontend/cypress/videos
+
+# Ignore IDE config files
+.vscode/*
```

Comment:
Please remove this entry. The files under this directory are being used by developers who use vscode, mostly for linting.

Reply:
Normally I add IDE specific config files to my gitignore, because every developer uses his own configs/settings. I removed this line.

Reply:
You are right. We can discuss it as a separate issue to remove .vscode directory from gitignore since it is not related to this PR.